### PR TITLE
INHEART-115: Add routing with sample sign in and homepage pages

### PR DESCRIPTION
**Pull request by krysia1**
- added react-native-router library
- added pages folder with Hompage and Sig In components
- moved App.tsx content to Homepage.tsx

### CODE REVIEW

**Review by BeataKula — APPROVED** [review]

File: apps/mobile/src/app/pages/homepage/homepage.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render } from '@testing-library/react-native';
+
+import Homepage from './homepage';
```
Comment (konradhanus) [inline apps/mobile/src/app/pages/homepage/homepage.spec.tsx:4]: - [] chanage folder homepage to Homepage start with capital letter

File: apps/mobile/src/app/pages/page-signin/sign-in.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { render } from '@testing-library/react-native';
+
+import SignIn from './sign-in';
```
Comment (konradhanus) [inline apps/mobile/src/app/pages/page-signin/sign-in.spec.tsx:3]: - [ ] - change name of sign-in.tsx to file to index.tsx then import will be shorter

File: apps/mobile/src/app/pages/page-signin/sign-in.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { SafeAreaView, Text } from 'react-native'
```
Comment (konradhanus) [inline apps/mobile/src/app/pages/page-signin/sign-in.tsx:1]: - [ ] - change folder name of page-signin start with capital letter without word page. -> Signin

File: apps/mobile/src/app/pages/page-signin/sign-in.tsx [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+import React from 'react';
+import { Link } from 'react-router-native';
+
+/* eslint-disable-next-line */
```
Comment (konradhanus) [inline apps/mobile/src/app/pages/page-signin/sign-in.tsx:5]: - [ ] - remove unused interface

File: apps/mobile/src/app/pages/page-signin/sign-in.tsx [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+export interface SignInProps {}
+
+export function SignIn(props: SignInProps) {
```
Comment (konradhanus) [inline apps/mobile/src/app/pages/page-signin/sign-in.tsx:8]: `export function SignIn() {`

**Review by konradhanus — APPROVED** [review]

**Pull request merged**